pytrip/tripexecuter/kernel.py

```diff
@@ -20,32 +20,32 @@
 Object holding all data needed for a specific projectile/rifi configuration.
 """
 
+from pytrip.tripexecuter.projectile import Projectile
+
 
 class KernelModel(object):
     """
     Kernel data, holding all necessary data for a certain projectile
     """
-    projectile = None
-    rifi_name = ""  # name of ripple filter, e.g. "3 mm Rifi", for documentation only
-    rifi_thickness = 0.0  # thickness of RiFi in mm
-    ddd_path = ""  # "/local/trip/data/ddd/*"
-    spc_path = ""  # "/local/trip/data/spc/*"
-    sis_path = ""  # if none, it will be autogenerated
-    comment = ""  # arbitrary text
-    name = ""  # name for identifying this kernel, ie.e "C12_3mmRiFi"
-
-    def __init__(self, name=""):
+    def __init__(self, projectile: Projectile, name: str = ""):
         """
         """
-        self.name = name
+        self.projectile = projectile
```

Comment:
Why do you move the properties from a class level to object level ?

Reply:
class level means static. Those variables arent static

Reply:
In python you don't have true statis variables, see caveats described here:
https://stackoverflow.com/questions/68645/are-static-class-variables-possible-in-python

I especially like this one:
```
>>> class MyClass:
...     i = 3
...
>>> MyClass.i
3 
(...)
>>> m = MyClass()
>>> m.i = 4
>>> MyClass.i, m.i
>>> (3, 4)
```

Anyway I agree that in such case moving attributes to object level makes sense